src/OpenTelemetry.Sampler.AWS/CHANGELOG.md

```diff
@@ -17,6 +17,9 @@
 * Updated OpenTelemetry core component version(s) to `1.10.0`.
   ([#2317](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2317))
 
+* `AWSXRayRemoteSamplerBuilder.Build()` now returns `ParentBasedSampler` which
+  which is of type `Trace.Sampler` instead of `AWSXRayRemoteSampler`
+
```

Comment:
```suggestion
* `AWSXRayRemoteSamplerBuilder.Build()` now returns `ParentBasedSampler` which
  which is of type `Sampler` instead of `AWSXRayRemoteSampler`.
  ([#2188](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2188))

```